we would need to look what's up with the effects on tests

```
FAILED dandi/tests/test_dandiarchive.py::test_parse_api_url[EMBER-DANDI-SANDBOX:000029/draft-parsed_url14] - AssertionError: assert DandisetURL(i...on_id='draft') == DandisetURL(i...on_id='draft')
  
  Omitting 2 identical items, use -vv to show
  Differing attributes:
  ['instance']
  
  Drill down into differing attribute instance:
    instance: DandiInstance(name='ember-dandi-sandbox', gui='https://apl-setup--ember-dandi-archive.netlify.app', api='https://api-dandi.sandbox.emberarchive.org/api') != DandiInstance(name='ember-dandi-sandbox', gui='https://apl-setup--ember-dandi-archive.netlify.app/', api='https://api-dandi.sandbox.emberarchive.org/api')
    
    Omitting 2 identical items, use -vv to show
    Differing attributes:
    ['gui']
    
    Drill down into differing attribute gui:
      gui: 'https://apl-setup--ember-dandi-archive.netlify.app' != 'https://apl-setup--ember-dandi-archive.netlify.app/'
      - https://apl-setup--ember-dandi-archive.netlify.app/
      ?                                                   -
      + https://apl-setup--ember-dandi-archive.netlify.app
= 1 failed, 903 passed, 1 skipped, 6 deselected, 3 xfailed, 2 xpassed in 680.99s (0:11:20) =
```

seems like just a trailing `/` difference, unclear how change of depth in domain name effects that here....

--- a/dandi/cli/tests/test_instances.py
+++ b/dandi/cli/tests/test_instances.py
@@ -24,7 +24,7 @@ def test_cmd_instances(monkeypatch):
         "  gui: https://dandi.emberarchive.org\n"
         "ember-dandi-sandbox:\n"
         "  api: https://api-dandi.sandbox.emberarchive.org/api\n"
-        "  gui: https://apl-setup--ember-dandi-archive.netlify.app/\n"
+        "  gui: https://apl-setup--ember-dandi-archive.netlify.app\n"
         "linc:\n"
         "  api: https://api.lincbrain.org/api\n"
         "  gui: https://lincbrain.org\n"

--- a/dandi/consts.py
+++ b/dandi/consts.py
@@ -150,7 +150,7 @@ known_instances = {
     ),
     "ember-dandi-sandbox": DandiInstance(
         "ember-dandi-sandbox",
-        "https://apl-setup--ember-dandi-archive.netlify.app/",
+        "https://apl-setup--ember-dandi-archive.netlify.app",
         "https://api-dandi.sandbox.emberarchive.org/api",
     ),
 }

--- a/docs/source/cmdline/instances.rst
+++ b/docs/source/cmdline/instances.rst
@@ -30,7 +30,7 @@ Example output:
       gui: https://lincbrain.org
     ember-dandi-sandbox:
       api: https://api-dandi.sandbox.emberarchive.org/api
-      gui: https://apl-setup--ember-dandi-archive.netlify.app/
+      gui: https://apl-setup--ember-dandi-archive.netlify.app
     ember-dandi:
       api: https://api-dandi.emberarchive.org/api
       gui: https://dandi.emberarchive.org

